src/main/groovy/com/blackbaud/templates/project/RestProjectBuilder.groovy

```diff
@@ -72,7 +72,7 @@ class RestProjectBuilder {
     public RestProject build() {
         BasicProject basicProject = createBasicProject()
         RestProject restProject = new RestProject(basicProject)
-        restProject.initRestProject(disableAuthFilter, vsts)
+        restProject.initRestProject(disableAuthFilter, vsts == false)
```

Comment:
any reason to not update this class to use `aws` instead of negating `vsts`?

Reply:
i was just being lazy but will update that

Reply:
well that was easy - @Blackbaud-ChrisJenkins updated